app/src/main/res/layout/recent_bookmarks_header.xml

```diff
@@ -1,39 +1,38 @@
 <?xml version="1.0" encoding="utf-8"?>
+
 <!-- This Source Code Form is subject to the terms of the Mozilla Public
    - License, v. 2.0. If a copy of the MPL was not distributed with this
    - file, You can obtain one at http://mozilla.org/MPL/2.0/. -->
+
 <androidx.constraintlayout.widget.ConstraintLayout
     xmlns:android="http://schemas.android.com/apk/res/android"
     xmlns:app="http://schemas.android.com/apk/res-auto"
     android:layout_width="match_parent"
-    android:layout_height="wrap_content" >
+    android:layout_height="wrap_content">
 
     <androidx.appcompat.widget.AppCompatTextView
-        style="@style/Header20TextStyle"
         android:id="@+id/recentlySavedBookmarksHeader"
+        style="@style/Header20TextStyle"
         android:layout_width="wrap_content"
         android:layout_height="wrap_content"
+        android:layout_marginTop="40dp"
         android:contentDescription="@string/recently_saved_bookmarks_content_description"
```

Comment:
I missed this during the previous code review of the styles. Why do we need to add a content description for title that accessibility can read out for us already?

Reply:
Same as above, breaking into another issue

Reply:
Can't we just remove the `contentDescription` in this PR and then the problem is solved?